include/number_four.h

```diff
@@ -0,0 +1,18 @@
+#ifndef A_FOUR
+#define A_FOUR
+
+#include "algorithm.h"
+
+class NumberFour : public Algorithm {
+public:
+    NumberFour(uint64_t seed) : Algorithm(seed) {
+        this->maxValue = 1;
```

Comment:
maybe change maxValue to 10 to give a scalar value [0, 1) to match the rest of the algos. im not sure if a value outside [0,1) as output from ./prng would work with all visuals